src/pages/Charcuterie.js

```diff
@@ -279,6 +280,13 @@ export default () => {
         debounceEnabled
         throttleTime={1000}
       />
+      <NavigationButtons
+        firstButtonText="Back"
+        firstButtonHref="https://www.linkedin.com/in/kevin-zou/"
+        secondButtonText="Next"
+        secondButtonHref="https://www.linkedin.com/in/kevin-zou/"
+        autosaveTime="4:20 pm"
```

Comment:
After you make the autosave time optional, can you also add an example of that on this board?
also...
nice time :trollface:

Reply:
_thank you_

Reply:
08d4c9e